CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Changed
 - Updated class calls to accommodate the latest release 0.13.1 of SwissPaymentSlipPdf
-
+- Implemented the umlaut fix
+- Changed the output code block to facilitate sprintf syntax
```

Comment:
Since we are just talking about examples, it's not important enough to be so detailed about such minor changes.
Better leave it away.